service/src/main/java/bio/terra/cbas/dependencies/github/GitHubClient.java

```diff
@@ -1,60 +1,46 @@
 package bio.terra.cbas.dependencies.github;
 
-import bio.terra.cbas.config.GitHubConfiguration;
 import com.google.gson.Gson;
 import jakarta.ws.rs.client.Client;
 import jakarta.ws.rs.client.ClientBuilder;
 import jakarta.ws.rs.client.WebTarget;
 import jakarta.ws.rs.core.MediaType;
 import jakarta.ws.rs.core.Response;
+import org.json.JSONObject;
 import org.springframework.stereotype.Component;
 
 @Component
 public class GitHubClient {
   private static final String BASE_URL = "https://api.github.com";
   private final Client client;
-  private final String token;
   private final Gson gson;
 
-  public GitHubClient(GitHubConfiguration gitHubConfiguration) {
+  public GitHubClient() {
     this.client = ClientBuilder.newClient();
-    this.token = gitHubConfiguration.personalAccessToken();
     this.gson = new Gson();
   }
 
-  public RepoInfo getRepo(String organization, String repo) throws GitHubClientException {
+  public JSONObject getRepo(String organization, String repo, String token)
```

Comment:
CBAS imports various libraries that can convert JSONObject into java classes. Our imported libraries for other dependencies all use them behind the scenes. It might be worth a quick investigation into whether we can use one of those to convert the json we get back into a well defined structure instead of a json object which the call just has to "know" how to use.

Reply:
Luckily, this was able to be resolved with a `@SerializedName` annotation on the field.